core/metacling/src/TClingClassInfo.h

```diff
@@ -173,6 +175,7 @@ class TClingClassInfo final : public TClingDeclInfo {
    bool                 IsLoaded() const;
    bool                 IsValidMethod(const char *method, const char *proto, Bool_t objectIsConst, Longptr_t *offset, ROOT::EFunctionMatchMode mode = ROOT::kConversionMatch) const;
    int                  InternalNext();
+   cling::LookupHelper::DiagSetting LookupDiagnostics() const;
```

Comment:
I find the current name not descriptive enough.  What about `HasLookupDiagnostics()`?